### DISCUSSION TITLE
SVG Rendering: How do change board colours (make transparent etc)

### DISCUSSION THREAD

**Discussion by jsblack** [niklasf/python-chess#777] [Q&A]
Hi!

I’ve been reading up on the docs about the SVG rendering and I can’t seem to find how to make a piece (or all pieces) transparent and likewise with squares and other objects, such as attacking icons. 

Can anyone point me in the right direction? Ideally when I render a board position, I’d like the ability to control the alpha and colour of each piece and square etc. 

Cheers

Jesse

**Comment by niklasf** [comment]
Parts of this may be implemented in the future (like #774), but in general the SVG module is intended for quick and dirty visualizations, not a fully fledged rendering framework for chess diagrams.

**Reply by jsblack** [reply]
Thanks mate! I ended up just deleting the paths for each piece and effectively made it transparent. Ended up doing the job for my final product: https://twitter.com/JesseSBlack/status/1390618737175195650
